frontend/package.json

```diff
@@ -1,52 +1,77 @@
 {
-    "name": "final-space-api-frontend",
-    "version": "0.0.0",
-    "scripts": {
-        "docusaurus": "docusaurus",
-        "start": "docusaurus start",
-        "build": "docusaurus build",
-        "swizzle": "docusaurus swizzle",
-        "deploy": "docusaurus deploy",
-        "serve": "docusaurus serve"
-    },
-    "author": "Ashutosh K Singh",
-    "license": "BSD-3-Clause",
-    "dependencies": {
-        "@docusaurus/core": "2.0.0-alpha.65",
-        "@docusaurus/plugin-pwa": "2.0.0-alpha.65",
-        "@docusaurus/preset-classic": "2.0.0-alpha.65",
-        "@docusaurus/theme-live-codeblock": "2.0.0-alpha.39",
-        "@mdx-js/react": "^1.5.8",
-        "axios": "^0.20.0",
-        "clsx": "^1.1.1",
-        "react": "^16.8.4",
-        "react-dom": "^16.8.4"
-    },
-    "browserslist": {
-        "production": [
-            ">0.2%",
-            "not dead",
-            "not op_mini all"
-        ],
-        "development": [
-            "last 1 chrome version",
-            "last 1 firefox version",
-            "last 1 safari version"
-        ]
-    },
-    "engines": {
-        "node": "10.x"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/lelouchB/final-space-api.git"
-    },
-    "keywords": [
-        "api",
-        "final space"
+  "name": "final-space-api-frontend",
+  "version": "0.0.0",
+  "scripts": {
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "serve": "docusaurus serve",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix --ext .js,.jsx .",
+    "format": "prettier --write './**/*.{js,jsx,css,md,json}'"
```

Comment:
Hey this command is not working properly 
Can you fix this
`npm run format`

Reply:
Can you explain more on what isn't working properly

Reply:
This command 
` npm run format`

Does not run on  my machine

Does it run on yours?

Reply:
It runs correctly on my machine (output below). 

Does it shows any error on your machine?

```
% npm run format

> final-space-api-frontend@0.0.0 format /Users/admin/Desktop/final-space-api/frontend
> prettier --write './**/*.{js,jsx,css,md,json}'

.eslintrc.js 49ms
.prettierrc.json 4ms
babel.config.js 10ms
blog/2020-10-04-welcome.md 57ms
blog/2020-10-07-new-characters.md 14ms
blog/2020-10-07-pwa-support.md 9ms
blog/2020-10-08-episode-endpoint.md 28ms
blog/2020-10-08-images-consistent.md 12ms
blog/2020-10-10-location-endpoint-added.md 31ms
docs/about.md 28ms
docs/character.md 57ms
docs/episode.md 50ms
docs/intro.md 27ms
docs/location.md 40ms
docusaurus.config.js 31ms
package-lock.json 446ms
package.json 5ms
README.md 14ms
sidebars.js 3ms
src/css/custom.css 43ms
src/pages/Components/_styles.module.css 9ms
src/pages/Components/Body/_Body.js 11ms
src/pages/Components/Body/_Card.js 20ms
src/pages/Components/Body/_CardList.js 3ms
src/pages/Components/Body/_InfoCointainer.js 10ms
src/pages/Components/Head/_Head.js 13ms
src/pages/index.js 6ms
static/manifest.json 10ms
% 
```

Reply:
Yes, 
```
$ npm run format

> final-space-api-frontend@0.0.0 format D:\code\API\final-space-api\final-space-api\frontend
> prettier --write './**/*.{js,jsx,css,md,json}'

[error] No files matching the pattern were found: "'./**/*.{js,jsx,css,md,json}'".
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! final-space-api-frontend@0.0.0 format: `prettier --write './**/*.{js,jsx,css,md,json}'`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the final-space-api-frontend@0.0.0 format script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:

```

Reply:
I was able to reproduce the error by switching to a Windows machine (I usually code on Mac).

Can you pull this branch again and see if it works.